docker-compose.yml

```diff
@@ -1,52 +1,35 @@
 version: "2"
 services:
-  redis:
-    image: redis
-    restart: always
-  web:
-    image: mammooc/mammooc.org
-    environment:
-      - RAILS_ENV=production
-      - RACK_ENV=production
-      - PORT=3000
-      - PASSENGER_MAX_POOL_SIZE=3
-      - AUTO_START_SIDEKIQ=false
-      - FORCE_SSL=false
-      - DOMAIN=
-      - DATABASE_URL=
-      - REDIS_PROVIDER=
-      - SECRET_KEY_BASE=
-      - SMTP_PORT=
-      - SMTP_HOST=
-      - SMTP_USERNAME=
-      - SMTP_PASSWORD=
-      - SMTP_DOMAIN=
-      - SMTP_AUTHENTICATION=
-      - WITH_S3=
-      - AWS_ACCESS_KEY_ID=
-      - AWS_SECRET_ACCESS_KEY=
-      - AWS_REGION=
-      - S3_BUCKET_NAME=
-      - NEW_RELIC_LICENSE_KEY=
-      - COURSERA_CLIENT_ID=
-      - COURSERA_SECRET_KEY=
-      - FACEBOOK_CLIENT_ID=
-      - FACEBOOK_SECRET_KEY=
-      - GOOGLE_CLIENT_ID=
-      - GOOGLE_SECRET_KEY=
-      - GITHUB_CLIENT_ID=
-      - GITHUB_SECRET_KEY=
-      - LINKEDIN_CLIENT_ID=
-      - LINKEDIN_SECRET_KEY=
-      - TWITTER_CLIENT_ID=
-      - TWITTER_SECRET_KEY=
-      - WINDOWS_LIVE_CLIENT_ID=
-      - WINDOWS_LIVE_SECRET_KEY=
-      - AMAZON_CLIENT_ID=
-      - AMAZON_SECRET_KEY=
-    command: sh 'start.sh'
-    ports:
-      - "<Host Port>:<Docker port>"
-    links:
-      - redis
-    restart: always
+    db:
+      image: postgres
+      ports:
+        - "5432"
+      volumes_from: 
+        - dbvolume
+    dbvolume:
+      image: busybox
+      volumes:
+        - /var/lib/postgresql/data
+    redis:
+      image: redis
+      ports:
+        - "6379"
+    web:
+      extends:
+        file: docker-compose-template.yml
+        service: web
+      build: .
+      environment:
+        - RAILS_ENV=development
+        - RACK_ENV=development
+        - WITH_S3=false
+        - DATABASE_URL=postgres://db:5432/
+        - REDIS_PROVIDER=redis://redis:6379/
+      command: sh 'start.sh'
+      ports:
+        - "3000:3000"
```

Comment:
If you don't want to change the port, it is possible to use ` - "3000"`, which will use 3000 as port for both, the container and the host.